src/components/EventPreview.js

```diff
@@ -1,35 +1,258 @@
 import React from 'react';
-import ExternalLink from '../components/ExternalLink';
-import { thumbnail, footerLinks } from './styles';
-import ImageLoader from "./ImageLoader";
+import PropTypes from 'prop-types';
+import moment from 'moment-timezone';
```

Comment:
I would prefer [`tiny-time`](https://github.com/aweary/tinytime), if possible

Reply:
Sweet, shouldn't take me long to swap that out. Thanks!

Reply:
@kwelch Updated 893227b